app/assets/stylesheets/application.tailwind.css

```diff
@@ -81,6 +81,43 @@
     @apply after:content-[''] after:block after:absolute after:top-0.5 after:left-0.5 after:bg-white after:w-4 after:h-4 after:rounded-full after:transition-transform after:duration-300 after:ease-in-out;
     @apply peer-checked:bg-green-600 peer-checked:after:translate-x-4;
   }
+
+  .releases-notes-row {
+    @apply mb-12;
+  }
+  .releases-notes-details {
+    @apply text-gray-500 text-sm;
+  }
+  .releases-notes-details h2 {
+    @apply text-lg mt-3.5 mb-2.5;
+  }
+  .releases-notes-details ul {
+    @apply list-disc my-2.5 pl-4;
+  }
+  .releases-notes-details p {
+    @apply my-2.5;
+  }
+  .releases-notes-details a {
+    @apply underline;
+  }
+  .releases-notes-details .user-mention {
+    @apply font-bold no-underline;
+  }
+  .releases-notes-details details {
+    @apply my-3.5 rounded-xl;
+  }
+  .releases-notes-details details video {
+    @apply rounded-b-xl;
+  }
+  .releases-notes-details summary {
+    @apply flex items-center gap-x-1;
+  }
+  .releases-notes-details .dropdown-caret {
+    @apply content-none border-4 border-b-0 border-transparent border-t-gray-500 size-0 inline-block;
+  }
+  .releases-notes-details .octicon {
+    @apply inline-block overflow-visible align-text-bottom fill-current;
+  }
```

Comment:
Definitely a tricky one here.  I think this is a good opportunity for us to evaluate the usage of typography within the app.

Ideally, I think what we should probably consider here is an API like so:

```erb
<div class="prose prose--github-release-notes">
  <%= release_notes[:body].html_safe %>
</div>
```

We have `prose` as the "base" styling mechanism, and then `prose--github-release-notes` as a "specialization" where we can add styles for things like `.user-mention`, `.octicon`, and `.dropdown-caret` which don't exist anywhere else.

That way, in `tailwind.config.js`, we can do all of our "base" [prose overrides](https://github.com/tailwindlabs/tailwindcss-typography?tab=readme-ov-file#customizing-the-css):

```js
// SAMPLE, not actual styles

/** @type {import('tailwindcss').Config} */
module.exports = {
  theme: {
    extend: {
      typography: {
        DEFAULT: {
          css: {
            color: '#333',
            a: {
              color: '#3182ce',
              '&:hover': {
                color: '#2c5282',
              },
            },
          },
        },
      },
    },
  },
  plugins: [
    require('@tailwindcss/typography'),
    // ...
  ],
}
```

And then in our global Tailwind CSS file, we can apply the "specializations":

```css
.prose--github-release-notes {
  .octicon {
    @apply inline-block overflow-visible align-text-bottom fill-current;
  }

  .dropdown-caret {
    @apply content-none border-4 border-b-0 border-transparent border-t-gray-500 size-0 inline-block;
  }
}
```

So in summary, the changes required here would be:

1. Delete existing `.prose` class in Tailwind CSS file (it's not being used anywhere)
2. Add `.prose--github-release-notes` and all the "specializations" to the Tailwind CSS file
3. Apply all "base" prose overrides directly in `tailwind.config.js`
4. Move all non-typography styles directly to the ERB files (e.g. the `.release-notes-row`, etc.)

Let me know what you think!

Reply:
@mattia-malnis for those "base" prose styles for the Tailwind Config file, I worked with @justinfar to standardize this a bit and included it in the updated design file:

https://www.figma.com/design/lonJmVk3HYkwZoIO7xYP2w/Maybe-App-(Community)?node-id=3789-202&t=UY4qtUbF3hRvnpp1-0

Please note—this is an _example_ that we should try to be as "generic" as possible with when applying to those "base" styles in the Tailwind config.  These "base" styles should work with _any_ HTML we need to render.

Reply:
@zachgoll your solution is definitely better, I didn't know about this possibility with Tailwind! I reviewed the code, let me know if everything is ok